src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -13,42 +13,33 @@
 import org.apache.pdfbox.rendering.ImageType;
 import org.apache.pdfbox.rendering.PDFRenderer;
 import org.apache.pdfbox.tools.imageio.ImageIOUtil;
-import org.springframework.web.bind.annotation.GetMapping;
-
-import javax.imageio.ImageIO;
 import java.awt.image.BufferedImage;
 import java.io.File;
-import java.io.IOException;
 
 /**
  * The class implements methods to convert PDF files into image files
  *
  * @author Andy Bazualdo on 9/23/19.
- * @version 1.0
+ * @version v1.0
  */
-public class ImageConvert {
+public class ImageConvert implements IConvert{
 
     /**
-     //* Creates the Controller Model using Tesseract with the wrapper Tess4J.
-     //*
-     //* @param sourcePath for use it as file to extract the text.
-     //* @param lang language of the file and for using the data training.
-     //* @return String with the text of the image.
-     //* @throws To implement after this demo
+     *  Creates the Controller Model using Tesseract with the wrapper Tess4J.
+     * //*
+     * //* @param sourcePath for use it as file to extract the text.
```

Comment:
This line needs to be just * not *//*.

Reply:
Can you review the new commit with the suggestion changes?